### ISSUE TITLE
Completion for packages installed in the current virtualenv

### ISSUE THREAD

**Issue by spwx** [roxma/nvim-completion-manager#136]
When you create and activate a Python virtual environment and then install packages in the environment, NCM is unable to provide completion for those packages.

deoplete.nvim + deoplete-jedi are able to provide this functionality. Is there someway to get this to work in NCM?

**Comment by spwx** [comment]
Looks like this problem is larger than NCM.

**Comment by roxma** [comment]
> When you create and activate a Python virtual environment and then install packages in the environment, NCM is unable to provide completion for those packages.

If you activate virtuanenv before starting nvim, it should work.

**Comment by bbkane** [comment]
This doesn't seem to work with conda envs. Is there some support I'm missing or perhaps an option to point jedi toward the first python in the PATH?
If you prefer, I can open a new issue.
Thanks

**Comment by roxma** [comment]
@bbkane 

The jedi completion process in running on `g:python3_host_prog`. Set it to the correct python path if necessary.

**Comment by bbkane** [comment]
@roxma, that means I must install `neovim`, `jedi` and friends in each environment, correct? And there's no way to use a system python to run ncm to autocomplete a project environment python?

**Comment by roxma** [comment]
@bbkane 

> that means I must install neovim, jedi and friends in each environment, correct? And there's no way to use a system python to run ncm to autocomplete a project environment python?

I'm not familiar with conda envs. With envs created by `python -m venv`, I don't have to install jedi in each virtual env. It seems this is the feature implemented by jedi.

**Comment by roxma** [comment]
@bbkane 

Try `pip install --user neovim jedi`
